.github/workflows/build.yml

```diff
@@ -441,8 +446,19 @@ jobs:
         with:
           persist-credentials: false
 
-      - name: Check semver
-        uses: obi1kenobi/cargo-semver-checks-action@v2
+      # QUICK WORKAROUND to AVOID ISSUE with cargo-semver-checks & portable-atomic
+      # TODO: INVESTIGATE & RESOLVE ROOT CAUSE
+      - run: cargo remove portable-atomic -p ${{ env.MAIN_CRATE }}
```

Comment:
I think this needs a note that this quick workaround does slightly mutate the source build configuration. As I said elsewhere I don't think this is anything to be concerned about, just needs to be noted (I think).